### ISSUE TITLE
Request: Add support for rearranging screen arrangement for external monitors

### ISSUE THREAD

**Issue by cjthompson** [kasper/phoenix#174]
- Version: 2.5
- macOS: 10.12.3

With the 2016 Macbook, a common problem that I have is that when using external monitors using USB-C -> Displayport adapters, macOS doesn't remember which monitor is on the left/right.  They often swap sides based on the order in which macOS recognizes monitors (both newly connected and after resuming from sleep).

According to [these docs](https://developer.apple.com/library/content/documentation/GraphicsImaging/Conceptual/QuartzDisplayServicesConceptual/Articles/Overview.html), there are APIs available to create a new screen arrangement.

I would like to see those APIs exposed so that a script could be written to detect screen arrangement and rearrange them.

I'm not a Mac developer, but I'd be happy to try to investigate this more in the code if someone can give me a starting point.

**Comment by kasper** [comment]
This is definitely something we can look into implementing.

**Comment by charlespeters** [comment]
@kasper is it possible to move windows between screens in the current implementation?

**Comment by kasper** [comment]
@charlespeters Of course, you just need to calculate the position of the window using the other screen’s frame. There’s example configurations in the Wiki.
